app/mailers/tasks_mailer.rb

```diff
@@ -0,0 +1,15 @@
+class TasksMailer < ApplicationMailer
+  default template_path: 'mailers/tasks'
+
+  def task_completed(task, current_user)
+    @task = task
+    @user = current_user
+    watchers = @task.watchers.map {|watcher| watcher.email if watcher != current_user}
+    if watchers.present?
```

Comment:
this is not good, if this returns false then in controller u'll receive `NoMethodError #deliver_later for nil:NilClass`